pkg/report/writer.go

```diff
@@ -83,7 +83,7 @@ func (tw TableWriter) Write(results Results) error {
 // TODO: refactror and fix cyclometic complexity
 func (tw TableWriter) write(result Result) {
 	table := tablewriter.NewWriter(tw.Output)
-	header := []string{"Library", "Vulnerability ID", "Severity", "Installed Version", "Fixed Version"}
+	header := []string{"Library", "Vulnerability ID", "Severity", "Installed Version", "Fixed Version", "URL"}
```

Comment:
I think it's better to put the URL on the rightmost side. We need to add "URL" after "Title".

Reply:
OK fixed